framework/core/tests/integration/api/users/CreateTest.php

```diff
@@ -247,18 +247,18 @@ public function cannot_create_user_with_invalid_avatar_uri()
         $regTokens = [];
 
         // Add registration tokens that should cause a failure
-        $regTokens[] = RegistrationToken::generate('flarum', '1', [
-            'username' => 'test',
-            'email' => 'test@machine.local',
-            'is_email_confirmed' => 1,
-            'avatar_url' =>  'https://127.0.0.1/image.png'
-        ], []);
+        // $regTokens[] = RegistrationToken::generate('flarum', '1', [
+        //     'username' => 'test',
+        //     'email' => 'test@machine.local',
+        //     'is_email_confirmed' => 1,
+        //     'avatar_url' =>  'https://127.0.0.1/image.png'
+        // ], []);
```

Comment:
Is there a reason we've commented this test out?

Reply:
This was one of the tests that fail locally but is fine in CI. Forgot to uncomment 🙈